src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,41 @@
+package duke.task;
+
+import duke.TaskManager;
+
+public class Task extends TaskManager {
+    protected String description;
+    public boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        if (isDone) { // mark done task with X
```

Comment:
Maybe remove the comment here since it is redundant and the code explains itself well already.